# Generated by Django 4.1.3 on 2023-03-13 17:22

import core.manager
import django.contrib.auth.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('core', '0006_alter_usermodel_password'),
    ]

    operations = [
        migrations.AlterModelManagers(
            name='usermodel',
            managers=[
                ('objects', core.manager.UserModelManager()),
            ],
        ),
        migrations.RenameField(
            model_name='usermodel',
            old_name='otp_enabled',
            new_name='is_phone_verified',
        ),
        migrations.RemoveField(
            model_name='usermodel',
            name='otp_auth_url',
        ),
        migrations.RemoveField(
            model_name='usermodel',
            name='otp_base32',
        ),
        migrations.RemoveField(
            model_name='usermodel',
            name='otp_verified',
        ),
        migrations.AddField(
            model_name='usermodel',
            name='otp',
            field=models.CharField(default=1, max_length=6),
            preserve_default=False,
        ),
        migrations.AlterField(
            model_name='usermodel',
            name='id',
            field=models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID'),
        ),
        migrations.AlterField(
            model_name='usermodel',
            name='username',
            field=models.CharField(error_messages={'unique': 'A user with that username already exists.'}, help_text='Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.', max_length=150, unique=True, validators=[django.contrib.auth.validators.UnicodeUsernameValidator()], verbose_name='username'),
        ),
    ]
